tests/test-posts.php

```diff
@@ -130,8 +132,14 @@ public function test_untrash_post() { // depends on post meta
 
 		wp_untrash_post( $id );
 
+		if ( version_compare( $wp_version, '5.6', '>=' ) ) { // https://core.trac.wordpress.org/ticket/23022
+			$expected_status = 'draft';
+		} else {
+			$expected_status = 'private';
+		}
```

Comment:
```suggestion
		// https://core.trac.wordpress.org/ticket/23022
		$expected_status = version_compare( $wp_version, '5.6', '>=' ) ? 'draft' : 'private';
```

Please consider this one-liner.

Reply:
This is really a matter of taste. Even though I really like and use ternary operators, some people argue that the multi line version is more readable (which I agree, when the operation gets more complex). In this specific case, I really don't have a preference.

Reply:
> some people argue that the multi line version is more readable

Yes, I know those people. They are not given many talents.

Reply:
Very sad. 😢 😢 😢